.eslintrc.js

```diff
@@ -92,6 +92,10 @@ module.exports = {
 				selector: 'CallExpression[callee.name="withDispatch"] > :function > BlockStatement > :not(VariableDeclaration,ReturnStatement)',
 				message: 'withDispatch must return an object with consistent keys. Avoid performing logic in `mapDispatchToProps`.',
 			},
+			{
```

Comment:
This rule seems generic enough that we may consider add it in packages/eslint-plugin/configs/custom.js

Reply:
Yeah, I'd thought about it. And even as a proper rule, it wouldn't be too difficult to implement a fixer for it. I also don't love `no-restricted-syntax` for the sole reason that when someone chooses to disable it, it's very unclear to the next person what exactly is being disabled (`no-restricted-syntax` is a lot less meaningful than something like `wordpress/no-truthy-length-rendering`). Also also, it could be a reasonable suggestion to offer upstream to [eslint-plugin-react](https://github.com/yannickcr/eslint-plugin-react).

That all being said, I saw it as something of a process of being "promoted" from a very immediately actionable solution, to something which has proved its value as generally useful to be included as part of a custom rule or a default configuration.